repositories.json

```diff
@@ -84,7 +84,8 @@
     {"version": 2, "development": "2-dev", "release": "master"}
   ],
   "sdk-dart": [
-    {"version": 2, "development": "2-dev", "release": "master"}
+    {"version": 2, "development": "2-dev", "release": "master"},
```

Comment:
Shoudn't we have a `2-stable` branch and have `2-dev` released on it ?

Reply:
```suggestion
    {"version": 2, "development": "2-dev", "release": "2-stable"},
```